docs/tutorials/setting-up-python-pycharm.md

```diff
@@ -1,83 +1,71 @@
 ---
 title: Setting Up a Python Development Environment with PyCharm
 group: software-languages
-author: "@antonvh"
+author: "@antonvh, @fisherds"
 ---
 
 * Table of Contents
 {:toc}
 
 This tutorial shows how to set up a nice working environment for developing
-Python on Ev3dev. The platform used is a BrickPi, but most of it is usable
-for regular LEGO EV3 bricks.
+Python on Ev3dev.
 {:.lead}
 
-Ev3dev is a lean and modern system to run on the Raspberry Pi that powers the
-BrickPi. It's got multiple languages that work in a pretty similar way and
-make switching easy. Moreover, if you code in ev3dev on the BrickPi, people
-with a regular EV3 and Ev3dev will be able to run your code! Alas, it doesn't
-work the other way around as the BrickPi has a more limited set of motor
-commands.
+For this tutorial, I will suppose you have burned an SD disk, booted it, and [connected to your
+RPi/BrickPi/EV3][network].
 
-For this tutorial, I will suppose you have burned an SD disk, connected your
-RPi and BrickPi to an ethernet cable and booted it. If you have an EV3 brick
-without ethernet, [set up another networking connection first][network].
+## Log in to ev3dev
 
-[network]: /docs/tutorials
-
-## Log in to ev3dev ##
-
-Now you can either attach a screen and keyboard to it for the setup, or
-connect to it via ssh. I prefer the latter. My router has a DNS that registers
-hostnames. It allows me to start a Terminal (on Mac) or Putty (on Windows) and
-simply do:
-
-`ssh robot@ev3dev`
+Now you can either or connect your ev3dev device [via ssh][ssh] or attach a screen and keyboard to it
+if using RPi. To log in via ssh, start a Terminal (on Mac) or Putty (on Windows) and type:
 
-If you don't have a DNS try: `robot@ev3dev.local` or connect to it's IP
-address directly. eg. `robot@192.168.0.2`
+`ssh robot@ev3dev.local`
```

Comment:
On Windows, `ssh robot@ev3dev` should work most of the time. Since most users are on Windows, it would make sense to list this option first.